source/src/BNFC/Backend/Java/CFtoJLex15.hs

```diff
@@ -45,6 +45,10 @@ import BNFC.Utils (cstring)
 import BNFC.Backend.Common.NamedVariables
 import Text.PrettyPrint
 
+
+cf2jlex' str cf = cf2jlex str cf False
+cf2jflex' str cf = cf2jlex str cf True
+
```

Comment:
I still think this needs fixing. I suggest the following solution: reorder the argument of `cf2jlex` and use `cf2jlex True` and `cf2jlex False` instead.

Or, if you really want two different functions that's ok but give them names without quotes.